### DISCUSSION TITLE
(Overlay) Affichage de la musique

### DISCUSSION THREAD

**Discussion by ghost** [skarab42/marv#43] [Ideas]
Lire la cover (image temporaire) + le texte (fichier temporaire) ou lecture de LastFM pour transmettre dans notre overlay

**Comment by skarab42** [comment]
Déjà possible a l'aide du plugin https://github.com/pendo324/universal-np 
Il manque juste un tuto pour expliquer comment faire ;)

**Reply by skarab42** [reply]
Pas la cover par contre...

**Reply by ghost** [reply]
ok car moi j utilise le plugin [Tuna](https://obsproject.com/forum/resources/tuna.843/) sur OBS qui donne un fichier text et une cover.
Donc au moins je pourrais mettre mon fichier texte ^^

**Reply by ghost** [reply]
Et aussi petite note supplementaire

J'ai reussi a placer une image temporaire sur Marv, en faisant apparaitre ma cover de musique directement dans `AppData\Local\marv-nodejs\Data\upload\filesAppData\Local\marv-nodejs\Data\upload\files`

**Reply by skarab42** [reply]
Ha oui bien vus si le nom de l'image de la cover ne change pas c'est good !

**Reply by mirobane** [reply]
aussi possible via TUNA , tuto anglais ici : https://youtu.be/sMb14n0qTkQ?t=242
